### PR TITLE
Worked on the dynamic column width of stats table

### PR DESCRIPTION
**Pull request by Ankit-Keshari-Vituity**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/preview/DefaultPreviewCard.tsx [hunk 1]
```diff
@@ -271,9 +289,15 @@ export default function DefaultPreviewCard({
                     />
                     <EntityTitleContainer>
                         <Link to={url}>
-                            <EntityTitle onClick={onClick} $titleSizePx={titleSizePx}>
-                                {name || ' '}
-                            </EntityTitle>
+                            {previewType === PreviewType.HOVER_CARD ? (
+                                <CardEntityTitle onClick={onClick} $titleSizePx={titleSizePx}>
+                                    {'Test Demo test Demo Test Demo Test Demo Chareacter her' || name || ' '}
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/preview/DefaultPreviewCard.tsx:294]: oops! remove the test text here

**Review by chriscollins3456 — COMMENTED** [review]
Just two quick comments then should be good to go!

File: datahub-web-react/src/app/preview/DefaultPreviewCard.tsx [hunk 2]
```diff
@@ -73,6 +73,24 @@ const EntityTitle = styled(Typography.Text)<{ $titleSizePx?: number }>`
     }
 `;
 
+const CardEntityTitle = styled(Typography.Text)<{ $titleSizePx?: number }>`
```
[the remark this answers is not in the document]
Comment (Ankit-Keshari-Vituity) [reply datahub-web-react/src/app/preview/DefaultPreviewCard.tsx]: Checked and Fixed!!

File: datahub-web-react/src/app/preview/DefaultPreviewCard.tsx (hunk 1 above)
Comment (Ankit-Keshari-Vituity) [reply datahub-web-react/src/app/preview/DefaultPreviewCard.tsx]: Checked and Fixed!!

**Review by chriscollins3456 — APPROVED** [review]
nice!

**Pull request merged**